src/Nox.Docs/Extensions/ScribanTemplateExtensions.cs

```diff
@@ -47,6 +50,14 @@ private static ScriptObject CreateScriptObject(Dictionary<string, object> model)
         var scriptObj = new ScriptObject();
         scriptObj.Import(model, renamer: member => member.Name, filter: null);
 
```

Comment:
@dejobratic I think we could reuse the `NoxSolutionBridge` that is on Nox.Generator
I would be ok to add a dependency to Nox.Generator

Even this code   

```
 var scriptObj = new ScriptObject();
        scriptObj.Import(model, renamer: member => member.Name, filter: null);
```

it's also being used in the TemplateCodeBuilder.GenerateSourceCode we could probably add to NoxSolutionBridge (maybe rename this), and use in both sides

BTW we are creating a TemplateFileBuilder, in the feature we need to reassess if make sense the diverged path we are taking in both of the sides